core/src/test/resources/azure-gateway-ccm-v2-jumpgate-init.sh

```diff
@@ -26,6 +26,7 @@ export CCM_V2_AGENT_KEY_ID="agentKeyId"
 export CCM_V2_AGENT_CRN="agentCrn"
 export CCM_V2_AGENT_BACKEND_ID_PREFIX="agentCrn-"
 export IS_CCM_V2_JUMPGATE_ENABLED=true
+export START_CCM_V2_JUMPGATE_AGENT=false
```

Comment:
Is not this in conflict with `IS_CCM_V2_JUMPGATE_ENABLED=true` in the line above?

Reply:
not

Reply:
Alright 👍